plugins/processors/gpuattributes/internal/awsneuron_metric_checker.go

```diff
@@ -0,0 +1,46 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+package internal
+
+import (
+	"strings"
+)
+
+const (
+	CONTAINER_NEURON_CORE_METRIC_PREFIX   = "container_neuroncore_"
```

Comment:
Are these being used anywhere else?

Reply:
nope, should be removed after migrating to the regex you mentioned